### DISCUSSION TITLE
How to format datetime scalar in query?

### DISCUSSION THREAD

**Discussion by ryanking1809** [prisma/prisma#2290] [Q&A] [answered]
Say I had a query to find all `posts` before `now()`, how do I correctly format the DateTime Scalar?

```graphql
query {
  posts(where: {createdAt: {lt: now()}}) {
    id,
  }
}
```

Perhaps this is a Nexus, not a Prisma issue?

**Comment by ryands17** [comment, marked as answer]
Hey @ryanking1809 :wave: 

`createdAt` will accept arguments for data types that are either a `Date` or a `String`.

So you could query for posts in the following manner,

```graphql
query {
  posts(where: { createdAt: { lt: new Date() } }) {
    id,
  }
}
```

or you could pass a `timestamp` value as a `String` as well.

Let me know if that solves this!

**Reply by ryanking1809** [reply]
I believe that does the trick, thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
